packages/web-vite/src/components/LabelPickerModal.tsx

```
@@ -0,0 +1,247 @@
+  )
+
+  // Check if search query exactly matches an existing label (case-insensitive)
+  const exactMatch = allLabels?.some(
+    (label) => label.name.toLowerCase() === searchQuery.toLowerCase().trim()
+  )
+
+  // Show create option if there's a search query and no exact match
+  const showCreateOption = searchQuery.trim() && !exactMatch
+
+  return (
+    <div className="label-picker-modal-overlay" onClick={onClose}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)